lib/azure/armrest/wrapper/availability_set.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_wrapper'
+
+module Azure
+  module Armrest
+    class AvailabilitySet < Azure::ArmRest::BaseWrapper
```

Comment:
We don't really need to create these many physical files for wrapper classes. Take an example from https://github.com/rest-client/rest-client/blob/master/lib/restclient/exceptions.rb. We can maintain a constant table in the base class and generate wrapper classes by code